cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -173,7 +174,8 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         val repository = artifactory.repository(repoName)
         try {
             repository.delete(releaseName_build)
-        } catch (_: Exception) {
+        } catch (e: Exception) {
+            logger.error("Exception occurred while deleting repository item: ", e)
```

The current implementation contains an empty catch block, which violates the best practice of handling exceptions properly. Ignoring exceptions can lead to silent failures without providing any information for debugging. To address this, we insert a logging statement within the catch block. This is done using the `logger.error` method to log both the exception message and stack trace, utilizing a logging library already part of the project dependencies (`io.github.microutils:kotlin-logging`). This approach maintains the original code flow without interrupting functionality, and it provides visibility into errors that might occur during repository item deletion.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)